### PR TITLE
Kapa v1.6.4 realese notes

### PR DESCRIPTION
**Pull request by lwandzura**
Closes https://github.com/influxdata/docs-v2/issues/3826

Added Kapa 1.6.4 release notes

### CODE REVIEW

File: content/kapacitor/v1.6/about_the_project/releasenotes-changelog.md [hunk 1]
```diff
@@ -7,6 +7,17 @@ menu:
     name: Release notes
 ---
 
+## v1.6.4 [2022-03-15]
+
+### Features
+
+- Add `SASL` support to `Kafka` alerts.
```
Comment (kelseiv) [inline content/kapacitor/v1.6/about_the_project/releasenotes-changelog.md:14]: should mention `SASL` support in the [Kafka AlertNode doc](https://docs.influxdata.com/kapacitor/v1.6/nodes/alert_node/#kafka) or the [Kafka event handlers doc](https://docs.influxdata.com/kapacitor/v1.6/event_handlers/)?

and would this look different in an alert/config example?

**Review by kelseiv — APPROVED** [review]

File: content/kapacitor/v1.6/about_the_project/releasenotes-changelog.md (hunk 1 above)
Comment (lwandzura) [reply content/kapacitor/v1.6/about_the_project/releasenotes-changelog.md:14]: I am deferring to @docmerlin for answers :-)

Comment (docmerlin) [reply content/kapacitor/v1.6/about_the_project/releasenotes-changelog.md:14]: @kelseiv @lwandzura  definitely in the kafka event handlers doc, that is where the detailed docs should go.

File: content/kapacitor/v1.6/about_the_project/releasenotes-changelog.md [hunk 2]
```diff
@@ -7,6 +7,17 @@ menu:
...
+
+### Bug fixes
+
+- Fully deprecate DES based ciphers, RC4 based ciphers and TLS 1.1 ciphers. 
+- Update `Kafka` client to use the `Sarama` Go client library.
```
Comment (docmerlin) [inline content/kapacitor/v1.6/about_the_project/releasenotes-changelog.md:19]: line 19 isn't really a bug fix.
line 18 should be not just TLS 1.1 but anything TLS1.1 or older.

**Review by docmerlin — CHANGES_REQUESTED** [review]

**Comment by samhld** [comment]
@lwandzura I believe we need to add config docs: https://github.com/influxdata/docs-v2/issues/3870

I believe that belongs here: https://docs.influxdata.com/kapacitor/v1.6/event_handlers/kafka/

File: content/kapacitor/v1.6/about_the_project/releasenotes-changelog.md (hunk 2 above)
Comment (docmerlin) [reply content/kapacitor/v1.6/about_the_project/releasenotes-changelog.md:19]: Go ahead and remove it.  It's an internal implementation detail and not really customer facing.

**Pull request merged**